route/emitter_test.go

```diff
@@ -138,10 +169,20 @@ var _ = Describe("Emitter", func() {
 	})
 
 	Context("When the publisher returns an error", func() {
+		useIngress = true
```

Comment:
Usually we set these variables in `BeforeEach` or `JustBeforeEach`. Can you please move it there? 

Also the same thing for the next Context.

cc @mnitchev